src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.csproj

```diff
@@ -67,9 +67,6 @@
     <Reference Include="Microsoft.Azure.KeyVault.Core">
       <HintPath>..\..\..\packages\Microsoft.Azure.KeyVault.Core.1.0.0\lib\net40\Microsoft.Azure.KeyVault.Core.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.Authorization">
```

Comment:
Because there has existed the 'ProjectReference' about Authorization in this csproj in line 244 :
    <ProjectReference Include="..\..\..\Common\Commands.Common.Authorization\Commands.Common.Authorization.csproj">
      <Project>{24508e26-154d-47f1-80ee-439bf0710996}</Project>
      <Name>Commands.Common.Authorization</Name>
    </ProjectReference>